importScripts/sql/transformations/LibraryExportView.sql

```diff
@@ -1,59 +1,91 @@
+DROP FUNCTION IF EXISTS clean_spaces;
+CREATE FUNCTION clean_spaces(str TEXT)
```

Comment:
I didn't like doing this, but the lookup of cleaned donor information was failing without this step, and the species were lost in the meantime. Anyway this is only in the data loading database.